README.md

```diff
@@ -30,7 +30,17 @@ include ::profile_hardening
 
 ## Usage
 
-The goal is that no paramters are required to be set. The default paramters should work for most NCSA deployments out of the box.
+The goal is that no parameters are required to be set. The default parameters should work for most NCSA deployments out of the box.
+
+To set temporary or Ad-hoc sysctl settings related to security, you can follow this Hiera example:
+```yaml
+profile_hardening::sysctl::settings:
+  "kernel.perf_event_paranoid":
+    comment: "reference a ticket, CVE, etc."
+    ensure: "present"
+    target: "/etc/sysctl.d/96-kernel-perf_event_paranoid.conf"
+    value: 3
+```
 
 
 ## Dependencies
```

Comment:
Do we need to add the sysctl Puppet module here?

Reply:
Done! I did include in metadata.json, but good to include in the README.md as well.